### PR TITLE
Fix hold date format

### PR DESCRIPTION
**Pull request by johnwatson484**
https://eaflood.atlassian.net/browse/SFI-2024

### CODE REVIEW

File: app/format-date.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const moment = require('moment')
 
-const formatDate = (dateToFormat, currentDateFormat = 'YYYY-MM-DD') => {
+const formatDate = (dateToFormat, currentDateFormat = 'DD/MM/YYYY HH:mm') => {
```
Comment (marctemp) [inline app/format-date.js:3]: Would you be a dear and add a unit test module on this

Comment (johnwatson484) [reply app/format-date.js:3]: Absolutely!  Apologies, just assumed there were some already.  

The fact that this is a bug probably should have been a clue there wasn't...

Comment (marctemp) [reply app/format-date.js:3]: Bit confused between the tests, does `currentDateFormat` refer to the default or current date?

Comment (johnwatson484) [reply app/format-date.js:3]: It's the format the actual date is stored in

**Review by marctemp — APPROVED** [review]
LGTM

**Pull request merged**